server/models/search.js

```diff
@@ -0,0 +1,15 @@
+const googleIt = require('google-it');
+
+function search(query) {
+	const options = { noDisplay: true };
+	const results = googleIt({ options: options, query: query })
+		.then((results) => {
+			return results;
+		})
+		.catch((e) => {
+			console.error(e);
+		});
+	return results;
+}
+
+module.exports = search;
```

Comment:
we had to google what google-it was, very nice find. Wish we had used it.
simple script written very nicely